src/app/work-item/work-item-quick-add/work-item-quick-add.component.scss

```diff
@@ -87,6 +74,70 @@
         }
       }
     }
+    .icon-btn, .icon-btn-disabled{
+      font-size: em(24);
+      text-decoration: none;
+    }
+    .icon-btn{
+      color: $color-pf-blue-400;
+      cursor: pointer;
+    }
+    .icon-btn-disabled{
+      color: $color-pf-black-300;
+      cursor: default;
+    }
+  }
+}
+
+.quickAdd_container2{
```

Comment:
Combine this class with **quickAdd_Container** and change the name to **quick-add-container**